src/common/oclengine.cpp

```diff
@@ -41,6 +41,31 @@ DeviceContextPtr OCLEngine::GetDeviceContextPtr(const int& dev)
     }
 }
 
+const std::vector<OCLKernelHandle> OCLEngine::kernelHandles = { OCLKernelHandle(OCL_API_APPLY2X2, "apply2x2"),
```

Comment:
```
#define OCL_HANDLE(X, x) OCLKernelHandle(OCL_API_##X, x)

const std::vector<OCLKernelHandle> OCLEngine::kernelHandles = {
    OCL_HANDLE(APPLY2X2, "apply2x2"),
    OCL_HANDLE(APPLY2X2_UNIT, "apply2x2_unit"),
    ...
};
#undef OCL_HANDLE
```

One definition per line, even if you have to disable auto-formatting around the block.

Reply:
Done.